### PR TITLE
final core coin changes

### PR DESCRIPTION
**Pull request by error2215**

### CODE REVIEW

File: clients/go-core/gocore.sh [hunk 1]
```diff
@@ -116,15 +123,15 @@ fi
 
 # Configure any mining operation
 if [ "$HIVE_MINER" != "" ] && [ "$HIVE_NODETYPE" != "light" ]; then
-    FLAGS="$FLAGS --mine --miner.coinbase $HIVE_MINER"
+    FLAGS="$FLAGS --mine --miner.corebase $HIVE_MINER --miner.threads=99"
```
Comment (OleksandrZeziulinskyi) [inline clients/go-core/gocore.sh]: Are you sure we can handle 99 threads?

File: simulators/smoke/mining/mining.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func main() {
 		},
 		Parameters: hivesim.Params{
 			"HIVE_PRIVATEKEY": "44cc42018b039b182dc1f8a05f4696995e39ba0e2af6d14f8dc68ee2fdf77195a85cbd6f8cf94905015b039300f44d8d639b287cb15abe4db0",
-			"HIVE_MINER":             "cb65e49851f010cd7d81b5b4969f3b0e8325c415359d",
+			"HIVE_MINER":             "ce56e49851f010cd7d81b5b4969f3b0e8325c415359d",
 			"HIVE_NETWORK_ID":            "1334",
```
Comment (OleksandrZeziulinskyi) [inline simulators/smoke/mining/mining.go:24]: Is HIVE_NETWORK_ID 19763 or 1334?

File: clients/go-core/gocore.sh [hunk 3]
```diff
@@ -116,15 +123,15 @@ fi
...
 fi
 if [ "$HIVE_MINER_EXTRA" != "" ]; then
     FLAGS="$FLAGS --miner.extradata $HIVE_MINER_EXTRA"
 fi
 
-# Configure LES.
+# # Configure LES.
 if [ "$HIVE_LES_SERVER" == "1" ]; then
-  FLAGS="$FLAGS --light.serve 50 --light.nosyncserve"
+  FLAGS="$FLAGS --light.serve 80 --bttp "
```
Comment (OleksandrZeziulinskyi) [inline clients/go-core/gocore.sh:133]: http?

File: simulators/smoke/mining/mining.go (hunk 2 above)
Comment (error2215) [reply simulators/smoke/mining/mining.go:24]: the default hive's network id is 19763. but you can set any network id for client

File: clients/go-core/gocore.sh (hunk 1 above)
Comment (error2215) [reply clients/go-core/gocore.sh]: I've tested it locally and it worked fine but I can set number of threads = 5 for example

File: clients/go-core/gocore.sh (hunk 3 above)
Comment (error2215) [reply clients/go-core/gocore.sh:133]: no, it's `--bttp` flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`

Comment (error2215) [reply clients/go-core/gocore.sh:133]: no, it's `--bttp` "blocks-to-trusted-peers" flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`

Comment (error2215) [reply clients/go-core/gocore.sh:133]: no, it's `--bttp` "blocks-to-trusted-peers" flag. 
`  --bttp                              Broadcasting blocks at first to trusted peers and then to all connected peers`

**Review by SashaZezulinsky — APPROVED** [review]

**Pull request merged**